.travis.yml

```diff
@@ -37,4 +40,4 @@ before_install:
 install:
     - composer update $COMPOSER_FLAGS
```

Comment:
can you add `./vendor/bin/simple-phpunit install` in this section too, so that the installation output from simple-phpunit gets collapsed by default on Travis (currently, the installation is done by the actual run)